### PR TITLE
Add Schedule log.

### PR DESCRIPTION
**Pull request by HMEiding**

### CODE REVIEW

File: cognite/client/experimental/model_hosting/schedules.py [hunk 1]
```diff
@@ -22,6 +22,33 @@ class ScheduleCollectionResponse(CogniteCollectionResponse):
     _RESPONSE_CLASS = ScheduleResponse
 
 
+class LogEntry(CogniteResponse):
+    def __init__(self, internal_representation):
+        super().__init__(internal_representation)
+        item = self.to_json()
+        print("hey hey")
+        print(item)
```
Comment (erlendvollset) [inline cognite/client/experimental/model_hosting/schedules.py]: remove these print statements

File: cognite/client/experimental/model_hosting/schedules.py [hunk 2]
```diff
@@ -120,3 +147,17 @@ def delete_schedule(self, id: int) -> None:
         """
         url = "/analytics/models/schedules/{}".format(id)
         self._delete(url=url)
+
+    def get_log(self, id: int) -> ScheduleLog:
+        """Return schedule log by id.
+
```
Comment (erlendvollset) [inline cognite/client/experimental/model_hosting/schedules.py]: Add some more description here about the logs the user can expect to have returned. i.e. "Two logs, one for failed scheduled predictions and one for successful."

File: cognite/client/experimental/model_hosting/schedules.py (hunk 1 above)
Comment (HMEiding) [reply cognite/client/experimental/model_hosting/schedules.py]: Wow, that was sloppy...

Comment (erlendvollset) [reply cognite/client/experimental/model_hosting/schedules.py]: I do it all the time. We should actually add a check in Jenkins which scans the library for print()s.

**Pull request merged**